doc/source/dev/data_managers.rst

```diff
@@ -4,23 +4,39 @@ Data managers
 What are Data Managers?
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Data Managers are a special class of Galaxy tool which allows for the download and/or creation of data that is stored within `Tool Data Tables <https://galaxyproject.org/admin/tools/data-tables/>`_ and their underlying flat (e.g. ``.loc``) files. These tools handle e.g. the creation of indexes and the addition of entries/lines to the data table / ``.loc`` file via the Galaxy admin interface.
+Data Managers are a special class of Galaxy tool which allows for the
+download and/or creation of data that is stored within
+`Tool Data Tables <https://galaxyproject.org/admin/tools/data-tables/>`_
+and their underlying flat (e.g. ``.loc``) files. These tools handle
+e.g. the creation of indexes and the addition of entries/lines to the
+data table / ``.loc`` file via the Galaxy admin interface.
 
-Data Managers can be defined locally or installed through the Tool Shed.
+Data Managers can be defined locally or installed through the ToolShed.
```

Comment:
I think the official spelling is "Tool Shed", although we have not been very consistent.

Reply:
I noticed this later, both are being used. I'll change to `Tool Shed`.